src/spell.js

```diff
@@ -732,14 +734,16 @@ class Spell {
    * @param {Object} obj - key-value pairs to SET
    */
   $update(obj) {
-    this.sets = parseSet(this, obj);
     this.command = 'update';
+    this.sets = parseSet(this, obj);
     return this;
   }
 
-  $increment(name, by = 1) {
-    const { Model } = this;
-
+  $increment(name, by = 1, opts = {}) {
+    let { Model, silent = false } = this;
+    silent = opts.silent;
```

Comment:
what if `this.silent === true` and `opts.silent === undefined`? looks like `this.silent` will be overwritten with wrong value.